items.json

```diff
@@ -334,32 +334,53 @@
     "id": "crpg_ambassador_sword_t3",
     "name": "The Ambassador",
     "culture": "Neutral",
-    "type": "OneHandedWeapon",
-    "price": 77,
-    "weight": 1.69,
-    "tier": 0.18010284,
+    "type": "TwoHandedWeapon",
+    "price": 13914,
+    "weight": 2.9,
+    "tier": 9.96596,
     "requirement": 0,
     "flags": [],
     "weapons": [
       {
         "class": "OneHandedSword",
-        "itemUsage": "onehanded_block_shield_swing_thrust",
+        "itemUsage": "onehanded_block_rshield_swing_thrust",

```

Comment:
this is not a conversion if it stills has 1h mod. Please remove the sword from onhander in weapon_description

Reply:
@namidaka this is fixed in the most recent push